packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1540,4 +1541,46 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
 
     return getOptionStyles(this.props.theme!, customStylesForAllOptions, customStylesForCurrentOption);
   }
+
+  /**
+   * Get the aria-activedescendant value for the comboxbox.
+   * @returns the id of the current focused combo item, otherwise the id of the currently selected element, null otherwise
+   */
+  private _getAriaActiveDescentValue(): string | null {
+    let descendantText = (this.state.isOpen && (this.state.selectedIndex as number) >= 0 ? (this._id + '-list' + this.state.selectedIndex) : null);
+    if (this.state.isOpen && this.state.focused && this.state.currentPendingValueValidIndex !== -1) {
+      descendantText = (this._id + '-list' + this.state.currentPendingValueValidIndex);
+    }
+    return descendantText;
+  }
+
+  /**
+  * Get the aria autocomplete value for the Combobox
+  * @returns 'inline' if auto-complete automatically dynamic, 'both' if we have a list of possible values to pick from and can
+  * dynamically populate input, and 'none' if auto-complete is not enabled as we can't give user inputs.
+  */
+  private _getAriaAutoCompleteValue(): string {
+    let autoComplete = !this.props.disabled && this.props.autoComplete === 'on';
+    return autoComplete ? (this.props.allowFreeform ? 'inline' : 'both') : 'none';
+  }
+
+  /**
+   * Get the string we would use for the title.
+   * @returns in order of priority:
+   * 1. The provided title
+   * 2. The label value that will be used by aria-labelled
+   * 3. The aria-label value
+   * 4. Else empty string
+   */
+  private _getTitle(): string {
+    let title = '';
+    if (this.props.title) {
+      title = this.props.title;
+    } else if (this.props.label) {
+      title = this.props.label;
+    } else if (this.props.ariaLabel) {
+      title = this.props.ariaLabel;
+    }
+    return title;
```

Comment:
We should only be setting the title if one was passed in, so instead of having _getTitle you can just use the title if it's given (inside of render)